sycl/test/basic_tests/buffer/buffer_dev_to_dev.cpp

```diff
@@ -4,6 +4,9 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 
+// Unsupported: cuda
+//  The test fails sporadically on cuda.
+
```

Comment:
```suggestion
// UNSUPPORTED: cuda
// The test fails sporadically on cuda. See https://github.com/intel/llvm/issues/1508 for more details.

```
This particular change has nothing to do with the "rounding modes for non-host devices", so I suggest sending it in a separate PR.

Reply:
FYI: I've created a separate PR with this change: https://github.com/intel/llvm/pull/1526.